README.md

```diff
@@ -496,6 +496,10 @@ AVALANCHEGO_PLUGIN_PATH="${HOME}/go/src/github.com/ava-labs/avalanchego/build/pl
 
 curl -X POST -k http://localhost:8081/v1/control/start -d '{"execPath":"'${AVALANCHEGO_EXEC_PATH}'","numNodes":5,"logLevel":"INFO","pluginDir":"'${AVALANCHEGO_PLUGIN_PATH}'","customVms":{"subnetevm":"/tmp/subnet-evm.genesis.json"}}'
 
+# or to expose local RPC server to all traffic
```

Comment:
can we made the example but avoiding the custom VM options. just the minimal options for the desired behaviour